schedule/migration/migration_offline_x86_64.yaml

```diff
@@ -0,0 +1,36 @@
+name:           migration_offline_x86_64.yaml
+description:    |
+  This is for offline migraiton tests on x86_64.  
```

Comment:
CI: Blank space at the end of line need remove.

Reply:
yes, thanks.